load_tests/create_testing_resources/kinesis_s3_firehose/resource_resolver.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
as this is the resource resolver for all testing resources. I think we can put it under folder `/create_testing_resources` directly

Reply:
Unfortunately, I couldn't get that to work. In python it is very difficult to import from a parent directory...